<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookMarkWindow</class>
 <widget class="QWidget" name="BookMarkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BookMarkWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="bookMarkList"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openGraphBtn">
       <property name="text">
        <string>打开图谱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delFromBookMarkBtn">
       <property name="text">
        <string>从书签中删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
